app/controllers/deliveries/show/recall.js

```diff
@@ -0,0 +1,14 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend({
+  actions: {
+    back() {
+      this.transitionToRoute('deliveries.show', this.get('model'));
+    },
+    recallDelivery() {
+      this.get('model.delivery')
+        .then(delivery => delivery.cancel())
+        .then(() => this.transitionToRoute('deliveries.show', this.get('model')));
```

Comment:
It would be nice to add an `infoMessage` param to this transition, like we do after sending or resending a delivery.

https://github.com/Duke-GCB/datadelivery-ui/blob/25f3e590cc03c3c799a10106a7e4846ad3e6f2ea/app/controllers/deliveries/show/resend-confirm.js#L41-L44